lib/llvm-ast.js

```diff
@@ -117,9 +128,13 @@ export class LlvmAstParser {
                     // This is a system header or implicit definition,
                     // remove everything up to the next top level decl
                     // Top level decls with invalid sloc as the file don't change the most recent file
-                    const slocRegex = /<<invalid sloc>>/;
-                    if (!slocRegex.test(output[i].text)) {
+                    if (systemSource.test(output[i].text)) {
+                        // skip ast from this source
+                    } else if (userSource.test(output[i].text)) {
+                        continue;
```

Comment:
codecov complains that no test exercises this path, but I expected at least some lines would, no?

Reply:
that might be so, I probably only tested it in the browser, will see if I can provide the test